packages/nameguard-react/src/components/Report/Check.tsx

```diff
@@ -93,7 +98,10 @@ function icon(code: CheckResultCode): React.ReactNode {
     default:
       return (
         <svg
-          className="w-[12px] h-[10px] fill-current text-gray-400 hover:text-gray-500 transition cursor-pointer"
+          className={cc([
+            "w-[12px] h-[10px] fill-current transition cursor-pointer",
+            checkResultCodeTextColor(CheckResultCode.skip, true),
+          ])}
           viewBox="0 0 12 10"
           fill="none"
           xmlns="http://www.w3.org/2000/svg"
```

Comment:
Hi @lightwalker-eth, thanks for these ideas, I have acted on them and on all the ones above this one as well.

The one thing I did not act was on adding `CheckIcon` to `/rating-shields` since I am yet facing a build issue on website repo, related to NameGuard React bundling. Once I have this fixed I will update `/rating-shields` with all our important components variations and send it to your review and feedback 👍🏼👨🏼‍💻

Reply:
Hi @lightwalker-eth ! `CheckIcon` was included with its different states in `/rating-shields` page 😉